src/views/sec/l10n.json

```
@@ -6,11 +6,11 @@
   "sec.projectsTitle": "Overview",
   "sec.eligibilityTitle": "Eligibility",
   "sec.applyTitle": "How do I apply?",
-  "sec.applyDeadline": "The deadline for applying to the Scratch Education has been extended to March 1st, 2021",
+  "sec.applyDeadline": "The deadline for applying to the Scratch Education Collaborative has been extended to March 1st, 2021",
   "sec.applyButton": "Click here to apply",
   "sec.projectsIntro": "The Scratch Education Collaborative (SEC), with support from Google.org, is building a powerful network of organizations across the world",
-  "sec.projectsIntroBold": "focused on supporting learners from historically marginalized and underserved communities developing their confidence with creative computing.",
-  "sec.projectsIntro2": "Up to 10 organizations will be selected in the 2021 pilot year to expand on and support each other's work with historically underrepresented youth including Black, LatinX, and Indigenous Americans, learn from one another, and collaborate with members of the Scratch Foundation, The MIT Media Lab, and other global leaders in creative computing to develop best practices for implementing culturally sustaining creative computing with Scratch.",
+  "sec.projectsIntroBold": "focused on supporting learners from historically marginalized communities in developing their confidence with creative computing.",
+  "sec.projectsIntro2": "Up to 10 organizations will be selected in the 2021 pilot year to expand on and support each other's work with historically marginalized communities including Black, Latinx, and Indigenous youth. Participants in the SEC will learn from one another, and collaborate with members of the Scratch Foundation, The MIT Media Lab, and other global leaders in creative computing to develop best practices for implementing culturally sustaining creative computing with Scratch.",
```

This is just a preference, but I'd probably put a comma there. Up to you!
```suggestion
  "sec.projectsIntro2": "Up to 10 organizations will be selected in the 2021 pilot year to expand on and support each other's work with historically marginalized communities, including Black, Latinx, and Indigenous youth. Participants in the SEC will learn from one another, and collaborate with members of the Scratch Foundation, The MIT Media Lab, and other global leaders in creative computing to develop best practices for implementing culturally sustaining creative computing with Scratch.",
```